scripts/push_docs_to_branch.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -Eeuo pipefail
+
+TMP_DOCS_FOLDER=tmp-documentation
+TODAY=`date "+%Y%m%d"`
+
+# use a temp directory, clone the $DOCS_BRANCH in it
+echo "Cloning $GITHUB_REPOSITORY#$DOCS_BRANCH into $TMP_DOCS_FOLDER..."
+mkdir $TMP_DOCS_FOLDER
+git clone --depth=1 --branch $DOCS_BRANCH git@github.com:$GITHUB_REPOSITORY.git $TMP_DOCS_FOLDER
+
+echo "Updating the docs..."
+cp $DOCS_FOLDER/* $TMP_DOCS_FOLDER/
+cd $TMP_DOCS_FOLDER
+
+if [ -z "$NEW_VERSION" ]
+then
+    echo "Skipping generation of new version."
+else
+    echo "Generating docs for new version $NEW_VERSION..."
+    yarn run new-version $NEW_VERSION
```

Comment:
this doesn't actually build the docs does it? it only creates a version, so not sure what we are pushing to git in this case either

Reply:
yes exactly, it creates a new version. Then Netflify will run the build command by itself.

Reply:
wait, so does the `documentation` branch contain rendered HTML pages or `mdx` pages?

Reply:
mdx pages. Otherwise we would have to version minified JS and HTML :/

Reply:
I've updated the PR desc, it should be clearer :)